src/components/pages/comparison.js

```diff
@@ -0,0 +1,315 @@
+        isDragged={false}
+        isEditing={false}
+        isPending={false}
+        dragHandleProps={{}}
+        canHardDelete={false}
+        context={SECTION_CONTEXT}
+        isOtherSectionDragged={false}
+        onSectionDelete={NOOP}
+        onSectionMoveUp={NOOP}
+        onSectionMoveDown={NOOP}
+        onSectionEditLeave={NOOP}
+        onSectionEditEnter={NOOP}
+        onSectionDuplicate={NOOP}
+        onSectionHardDelete={NOOP}
+        onPendingSectionApply={NOOP}
+        onSectionContentChange={NOOP}
+        onPendingSectionDiscard={NOOP}
+        onSectionCopyToClipboard={NOOP}
+        />
+    );
+  };
+
+  const renderSectionMissingInfo = () => {
+    return (
+      <div className="ComparisonPage-sectionMissingInfo">
+        {t('nonExistentSection')}
+      </div>
+    );
+  };
+
+  const renderSectionMovementInfo = sectionComparison => {
+    const handleFollowSectionMovementClick = event => {
+      event.preventDefault();
+      const targetElement = window.document.getElementById(sectionComparison.targetKey);
+      targetElement?.scrollIntoView({ behavior: 'smooth', block: 'center' });
+    };
+    return (
+      <div id={sectionComparison.key} className="ComparisonPage-sectionMovementInfo" key={sectionComparison.key}>
+        <Trans
+          t={t}
+          i18nKey="sectionMovementInfo"
+          values={{ placement: sectionComparison.changeType }}
+          components={[<a key="link" onClick={handleFollowSectionMovementClick} />]}
+          />
+      </div>
+    );
+  };
+
+  const renderSectionMovementBackInfo = sectionComparison => {
+    if (sectionComparison.changeType === SECTION_CHANGE_TYPE.movedHere) {
+      const handleFollowSectionMovementBackClick = event => {
+        event.preventDefault();
+        const targetElement = window.document.getElementById(sectionComparison.targetKey);
+        targetElement?.scrollIntoView({ behavior: 'smooth', block: 'center' });
+      };
+      return (
+        <div className="ComparisonPage-sectionMovementBackInfo">
+          <Trans
+            t={t}
+            i18nKey="sectionMovementInfoBack"
+            components={[<a key="link" onClick={handleFollowSectionMovementBackClick} />]}
+            />
+        </div>
+      );
+    }
+
+    return null;
+  };
+
+  const renderMetadataComparison = metadataComparison => {
+    const panelContent = (
+      <div className="ComparisonPage-metadataPanelContent">
+        <DiffView diff={metadataComparison.diff} />
+      </div>
+    );
+
+    return renderPanel({ id: 'metadata', info: `${t('metadata')}`, actions: [], content: panelContent });
+  };
+
+  const renderSectionComparison = sectionComparison => {
+    if (sectionComparison.changeType === SECTION_CHANGE_TYPE.movedDown || sectionComparison.changeType === SECTION_CHANGE_TYPE.movedUp) {
+      return renderSectionMovementInfo(sectionComparison);
+    }
+
+    const displayMode = displayModesByMappedRevisionKey[sectionComparison.key];
```

Comment:
I'm not sure if it's this `key` or another, that causes a react warning of "two children with the same key" when previewing a section that contains changes. Does not occur for when the section was added or deleted.

Reply:
Good find, thanks. Fixed!